apC.md

```diff
@@ -64,23 +64,23 @@ printIf( isLongEnough, msg1 );
 printIf( isLongEnough, msg2 );			// Hello World
 ```
 
-A few differences to point out compared to Chapter 3's approach:
+与我们在第 3 章中的实现相比有几处不同：
 
-* We use `R.complement(..)` instead of `not(..)` to create a negating function `isLongEnough(..)` around `isShortEnough(..)`.
+* 使用 R.complement(..) 代替 not(..) 函数来创建 isShortEnough(..) 的含义对立函数 isLongEnough(..).
```

Comment:
1、`R.complement(..)` markdown 语法请保留
2、instead of： 而不是
3、使用 R.complement(..) 代替 not(..) 函数来创建 isShortEnough(..) 的含义对立函数 isLongEnough(..). => 我们使用 `R.complement(..)` 而不是 `not(..)` 在 `isShortEnough(..)` 周围新建一个否定函数 `isLongEnough(..)`